web/wp-content/themes/dctx/archive.php

```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-k4yl3w.php, but found phpcs-scan-k4YL3w.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).